spec/board_spec.rb

```diff
@@ -0,0 +1,235 @@
+require 'spec_helper'
+require_relative '../lib/board'
+
+describe Board do
+  describe 'board_string' do
+    it 'will print a blank board when the board array contains only empty strings' do
```

Comment:
Nitpicky comment: How about `it 'prints a blank board...` (the `it` is the assertion block, so will is implied)

And now that I think about it, you're not actually testing that something "prints", you're testing what is returned. So it should probably be something like `it 'returns a blank board string...'`